cfme/v2v/migrations.py

```diff
@@ -166,16 +166,16 @@ def fill(self, values):
                         ...
                     }
         """
-        for cluster in values:
+        for cluster in (values if isinstance(values, dict) else []):
             if self.cluster_selector.is_displayed:
                 self.cluster_selector.fill(cluster)
             for mapping in values[cluster]['mappings']:
                 self.source_datastores.fill(mapping['sources'])
                 self.target_datastores.fill(mapping['target'])
                 self.add_mapping.click()
         was_change = not self.mappings_tree.is_empty
-        if was_change:
-            self.logger.info("Fill operation was successful, click Next.")
+        self.cancel_btn.wait_displayed()
+        if self.next_btn.is_displayed:
```

Comment:
What happens if the next button is not changed here?

Reply:
Removed this line . next button is always displayed